applicationinsights-android/src/main/java/com/microsoft/applicationinsights/internal/Sender.java

```diff
@@ -104,7 +106,7 @@ private void sendRequestWithPayload(String payload) throws IOException {
             InternalLogging.error(TAG, e.toString());
             Persistence persistence = Persistence.getInstance();
             if (persistence != null) {
-                persistence.persist(payload);
+                persistence.persist(payload, false); //TODO we should differentiate between crashes and telemetry here
```

Comment:
Do we actually need to make the differentiation or is it sufficient to flag a batch of telemetry that contains a crash as high-priority?